hadoop-hdds/docs/content/feature/PrefixFSO.md

```diff
@@ -62,32 +66,36 @@ Following picture describes the OM metadata changes while performing a rename
 {{< image src="PrefixFSO-Rename.png">}}
 
 ## Configuration
-By default the feature is disabled. It can be enabled with the following
- settings in `ozone-site.xml`:
+
+The following configuration can be used to define the default value for bucket layout during bucket creation
+if the client has not specified the bucket layout argument.
+Supported values are `OBJECT_STORE` and `FILE_SYSTEM_OPTIMIZED`.
+
+By default, the buckets will default to `OBJECT_STORE` behaviour.
 
 ```XML
+
 <property>
-   <name>ozone.om.enable.filesystem.paths</name>
-   <value>true</value>
-</property>
-<property>
-   <name>ozone.om.metadata.layout</name>
-   <value>PREFIX</value>
+    <name>ozone.default.bucket.layout</name>
+    <value>OBJECT_STORE</value>
 </property>
 ```
 
-In reference to efforts towards supporting protocol aware buckets 
-within an Ozone cluster, the following configuration can be used 
-to define the default value for bucket layout during bucket creation 
-if the client has not specified the bucket layout argument. 
-Supported values are `OBJECT_STORE` and `FILE_SYSTEM_OPTIMIZED`.
+## Configuring Older Versions (1.2.1)
 
-By default, the buckets will default to `OBJECT_STORE` behaviour.
+Older clients which do not support bucket layout features can enable cluster-wide prefix-based file system optimization
+using the following configuration.
+
+By default, the feature is disabled. It can be enabled with the following settings in `ozone-site.xml`:
 
 ```XML
 
 <property>
- <name>ozone.default.bucket.layout</name>
- <value>OBJECT_STORE</value>
+    <name>ozone.om.enable.filesystem.paths</name>
+    <value>true</value>
+</property>
+<property>
```

Comment:
Since the feature is not released yet we can remove the` cluster-wide prefix-based file system optimization` point and `ozone.om.metadata.layout` configuration.

Reply:
Thanks for the reviews @rakeshadr, I have removed references to `ozone.enable.filesystem.paths`.